### PR TITLE
Tickets/dm 29961

### PR DESCRIPTION
**Pull request by suberlak**
This updates the call to the `phosim_repackager.py` in `PhosimCmpt.py`   adding `focusz`  argument.

### CODE REVIEW

File: python/lsst/ts/phosim/PhosimCmpt.py [hunk 1]
```diff
@@ -1220,14 +1220,18 @@ def repackagePistonCamImgs(self, isEimg=False):
 
         intraFocalDirName = self.getIntraFocalDirName()
         extraFocalDirName = self.getExtraFocalDirName()
-        for imgType in (intraFocalDirName, extraFocalDirName):
-
+        defocalDistInMm = self.tele.defocalDist
```
Comment (teweitsai) [inline python/lsst/ts/phosim/PhosimCmpt.py:1223]: Please consider to use:
```
defocalDistInMm = self.tele.getDefocalDistInMm()
```

BTW, I do not think the `TeleFacade` class has the attribute of `defocalDist`.

**Review by teweitsai — APPROVED** [review]
This update looks good to me. However, the Jenkins test should fail because of the version of `ts_ofc`. Please try to checkout the `v1.3.7` of `ts_ofc` in Jenkinsfile to pass the test:
https://github.com/lsst-ts/ts_phosim/blob/develop/Jenkinsfile#L41-L42

Thanks!

**Review by tribeiro — CHANGES_REQUESTED** [review]
Instead of merging this, lets merge my PR. I did the exact same "fix" there.

**Comment by teweitsai** [comment]
I think we need to merge Tiago's ticket branch to `develop` branch first, then, rebase this ticket branch to `develop` branch and fix any failure in this PR.

**Comment by tribeiro** [comment]
I think we should close this PR. My PR #50 is already implementing it.

**Pull request closed without merging**